app/search_cmd.go

```diff
@@ -102,6 +103,15 @@ func (s *searchCmd) Run(l *ui.UI, env *hermit.Env, state *state.State) error {
 		}
 		return nil
 	}
+	if s.Exact {
```

Comment:
I think an alternative would be to add this before the search:

```go
pattern := s.Constraint
if s.Exact {
  pattern = "^" + regexp.QuoteMeta(pattern) + "$"
}
```

then pass `pattern` into the search.

Reply:
Maybe update`Constraint` too, rename it to `Pattern` and update the help to mention the both semantics.